pipelines/controller/src/main/resources/templates/index.html

```diff
@@ -404,6 +421,19 @@ <h2 class="accordion-header" id="flush-headingThree">
             </div>
           </div>
         </div>
+        <div style="margin-top: 50px">
+          <h3>Metrics</h3>
+          <div class="card md-5 rounded-3 shadow-sm">
+            <div class="card-header py-3">
+              Fetch the latest application and system metrics
+                <button type="submit" style="float: right; margin: -7;" class="button btn btn-primary" onclick="fetchLatestMetrics()">
```

Comment:
nit: I understand there are already some very long lines in this file, but please break new long lines when it is easy to do so.

Reply:
Done